helm-chart/crds/retrytimeoutconfig.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: retrytimeoutconfig.nsm.nginx.com
+  labels:
+    app.kubernetes.io/part-of: nginx-service-mesh
+spec:
+  group: specs.smi.nginx.com
+  scope: Namespaced
+  names:
+    kind: RetryTimeoutConfig
+    listKind: RetryTimeoutConfigList
+    shortNames:
+      - rtc
+    singular: retrytimeoutconfig
+    plural: retrytimeoutconfigs
+  versions:
+    - name: v1alpha1
```

Comment:
nit: for consistency with other lists in our YAML configs, remove the indent on this section. (same applies for any list in this config)

Reply:
My formatter and I agree that indented lists are more readable, but I un-indented them so they will be consistent with the other configs. 😄